bin/bpmn-auto-layout.ts

```diff
@@ -0,0 +1,17 @@
+#!node
+
+import * as fs from 'fs';
```

Comment:
Let us have this in javascript, not typescript.

Let us also use `#!/usr/bin/env node` as a shebang.